spec/features/admin_edits_category_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+feature 'admin edits category', %Q{
+  As an admin, I want to edit a category or subcategory, so that it better
+  represents the content under it.
+  Acceptance Criteria:
+  * [X] - I can edit the name of a category inline.
+  * [X] - I can edit the name of a subcategory inline.
+  } do
+
+  before(:each) do
+    @admin = FactoryGirl.create(:user, admin: true)
+    @category = FactoryGirl.create(:category, user: @admin)
+  end
+
+  scenario 'admin edits category title', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.